roles/ci_gen_kustomize_values/templates/bgp_dt01/network-values/values.yaml.j2

```diff
@@ -67,6 +67,13 @@ data:
         - end: {{ range.end }}
           start: {{ range.start }}
 {%    endfor %}
+{%    if network.tools.netconfig.ipv4_routes | default(false) %}
```

Comment:
Netconfig has no routes options yet
https://github.com/openstack-k8s-operators/ci-framework/blob/e9e103760350dbc675b9a03d8952aa0054f7fb07/plugins/module_utils/net_map/networking_env_definitions.py#L163

Reply:
@pablintino, thanks for your comment.
I have updated the PR, although I'm not sure if the solution is so simple.